inference.py

```diff
@@ -48,7 +48,7 @@ def run_inference(question, prompt_file="prompt.md", metadata_file="metadata.sql
             eos_token_id=eos_token_id,
             pad_token_id=eos_token_id,
         )[0]["generated_text"]
-        .split("```sql")[-1]
+        .split("# SQL\n")[-1]
```

Comment:
Hi @KimBioInfoStudio , I believe ```sql might be a better separator since those are the final characters in the prompt. If you're using a different prompt, you can edit this portion accordingly as you did but we won't merge that into our repo to ensure validity with the existing prompts.